src/SFML/Graphics/Font.cpp

```diff
@@ -731,7 +732,7 @@ IntRect Font::findGlyphRect(Page& page, unsigned int width, unsigned int height)
             continue;
 
         // Check if there's enough horizontal space left in the row
-        if (width > page.texture.getSize().x - it->width)
+        if (width > size.x - it->width)
```

Comment:
[`sf::Texture::getSize()` returns a precomputed variable](https://github.com/SFML/SFML/blob/a2626365908ded901fa5aef9ecf4d3dacbd03f47/src/SFML/Graphics/Texture.cpp#L310-L313), so the most you can win is an extra function call, which could be inlined anyway.

I would definitely not add this change for performance reasons, more for the sake of avoiding repetition. But then the variable should be named `pageTextureSize` or so for clarity, and then you have again almost the same verbosity as before. In addition, it may require the reader to look up the declaration of this variable further up to see what it actually does.

So: it _can_ be changed, not sure the pros outweight the cons though.

Reply:
@Bromeon fix it.
https://github.com/SFML/SFML/pull/1732/commits/ae2d032d5a33e7f63fb2355b64426eef870eb0e5